server/sonar-server/src/test/java/org/sonar/server/i18n/I18nRule.java

```diff
@@ -26,16 +26,33 @@
 import java.util.Map;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
 import org.sonar.api.i18n.I18n;
 
-public class I18nRule implements I18n {
+public class I18nRule implements TestRule, I18n {
```

Comment:
It's to allow usage of Rule annotation which allow to clean up i18 bundles between each test